test/quantization/core/test_quantized_tensor.py

```diff
@@ -984,6 +1010,37 @@ def test_qtensor_fill_per_channel(self):
             self.assertEqual(q_filled.q_per_channel_scales(), scales)
             self.assertEqual(q_filled.q_per_channel_zero_points(), zero_points)
 
+    # adapted from test_qtensor_fill_per_channel and test_qtensor_fill_per_tensor_nhwc
+    def test_qtensor_fill_per_channel_nhwc(self):
+        dims = torch.randint(low=1, high=10, size=(4, )).tolist()
+        axis = 0
+        # adding a constant to avoid too small of a scale
+        scales = torch.rand(dims[axis], dtype=torch.float64) + 0.1
+        zero_points = torch.randint(low=0, high=10, size=(dims[axis], ))
+
+        ones = torch.ones(dims).to(torch.float)
+
+        qtypes = [torch.qint8, torch.quint8, torch.qint32]
+        vals2fill = [-1, 1, 2**32]  # positive, negative, overflow
+        memory_formats = [torch.contiguous_format, torch.channels_last]
+        devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
```

Comment:
we have a util function here: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_tensor.py#L70, maybe we can move this function to common_quantization.py and import to this file, since it's repeated here as well: https://github.com/pytorch/pytorch/blob/master/test/quantization/fx/test_quantize_fx.py#L168

Reply:
will call this function and have a separate PR that refactors this to common_quantization